### PR TITLE
feat(sheet): auto height in row context menu 

### PR DESCRIPTION
**Pull request by Jocs**
- [x] Fit for data menu item
- [x] After adjusting the row height, set `isAutoHeight` to `false`
- [x] Add SetRowAutoHeightCommand
- [x] When calculating the automatic row height, determine whether the row `isAutoHeight` is `true`
- [x] Row height maximum 2000 limit
- [x] Add SetRowAutoHeightCommand test case

### CODE REVIEW

File: packages/base-sheets/src/commands/commands/__tests__/set-row-height.command.spec.ts [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+    RANGE_TYPE,
+    RedoCommand,
+    UndoCommand,
+    Univer,
+} from '@univerjs/core';
 import { Injector } from '@wendellhu/redi';
 import { afterEach, beforeEach, describe, expect, it } from 'vitest';
 
-import { SetWorksheetRowHeightMutation } from '../../..';
+import { SetWorksheetRowHeightMutation, SetWorksheetRowIsAutoHeightMutation } from '../../..';
```
Comment (wzhudev) [inline packages/base-sheets/src/commands/commands/__tests__/set-row-height.command.spec.ts:14]: 注意这里的引用路径

File: packages/ui-plugin-sheets/src/controller/menu/menu.ts [hunk 2]
```diff
@@ -47,6 +47,7 @@ import {
     SetWorksheetRowHeightCommand,
     SetWorksheetShowCommand,
 } from '@univerjs/base-sheets';
+import { SetRowIsAutoHeightCommand } from '@univerjs/base-sheets/commands/commands/set-worksheet-row-height.command.js';
```
Comment (wzhudev) [inline packages/ui-plugin-sheets/src/controller/menu/menu.ts:50]: 注意引用路径问题

这里要不写一下 eslint 规则，禁止在引入 univerjs 出现穿透子路径的行为

**Review by wzhudev — CHANGES_REQUESTED** [review]
Some minor issues need to be addressed.

File: packages/ui-plugin-sheets/src/controller/menu/menu.ts (hunk 2 above)
Comment (Jocs) [reply packages/ui-plugin-sheets/src/controller/menu/menu.ts:50]: OK

File: packages/base-sheets/src/commands/commands/__tests__/set-row-height.command.spec.ts (hunk 1 above)
Comment (Jocs) [reply packages/base-sheets/src/commands/commands/__tests__/set-row-height.command.spec.ts:14]: OK

File: packages/ui-plugin-sheets/src/controller/menu/menu.ts (hunk 2 above)
Comment (Jocs) [reply packages/ui-plugin-sheets/src/controller/menu/menu.ts:50]: 后面需要加一下规则，毕竟后面我们要分包发布？

**Review by wzhudev — APPROVED** [review]

**Pull request merged**